### PR TITLE
Update rocket script version for WPR

### LINKED ISSUE #6728: Largest element with a background-image that is not kept (such as linear-gradient) makes the OCI optimization have no LCP
**Context**
The issue is reported here: https://wp-media.slack.com/archives/C072P5EU5DF/p1718772180824549

Checking with WPR Inspector, I can see the beacon returns this to AJAX:
image

I added a JS file identical to the beacon in Chrome dev tools, added a log of `firstElementWithInfo` and found the LCP element was div.x-section.e9265-e1.m75d-0.m75d-1.hero-section.
It is indeed the largest element with a background-image.
But the background image is `linear-gradient(164deg, rgb(0, 0, 0) 0%, rgb(0, 65, 85) 53%, rgb(35, 213, 171) 97%)` which is not kept in the `else` branch of `_getElementInfo` (I think it does not passes the regex there).
As a result, the method `_getElementInfo` returns an element with an elementInfo, but it is empty. This empty element takes the place of LCP.

**Expected behavior**
TBC with Product team: I think we don't want to account for linear-gradient and instead, pick the next element as LCP. Is this correct?

**Acceptance Criteria**
TBC with Product team.

**Additional information**
If we want to skip elements with a background-image that is set but not as expected currently (for instance linaer-gradient), then I as able to get this behavior by just adding:

```
			if (element_info.bg_set.length <= 0) {
				return null;
			}
```

in `_getElementInfo` just before `if (element_info.bg_set.length > 0) {`

### LINKED ISSUE #6978: Exclude `svg use` targets from ALR
**Before submitting an issue please check that you’ve completed the following steps:**
- Made sure you’re on the latest version [x]
- Used the search feature to ensure that the bug hasn’t been reported before [x]

**Describe the bug**
SVG `use` might call a lazy rendered elements on the page. In such case, the SVG won't look as expected as it still inherits the skipped layout and painting work from the originally lazy rendered element. 

**To Reproduce**
Case discussed here - https://wp-media.slack.com/archives/C07LKLTKBG8/p1726606365110269

**Expected behavior**
Elements targeted by `svg use` shouldn't be lazy rendered. The exclusion should include any of their parents eligible for LRC. 

We don't know where exactly the target element is located. Example, the instructing `use` element might come from an element below the fold, but the target element is still further down on the page. Consequently, we should go with a systematic dismissal of targeted elements.

**Screenshots**
N/A

**Additional context**
https://wp-media.slack.com/archives/C07LKLTKBG8/p1726606365110269

**Acceptance Criteria (for WP Media team use only)**
Clear instructions for developers, to be added before the grooming

### LINKED ISSUE #7018: Wrong OCI if scroll happens before beacon fires - Update rocket-scripts dependencies
**Describe the bug**
Wrong OCI will be picked up if the page is scrolled before the beacon script runs. 
This issue is here to update the `rocket-scripts` dependency once it's been fixed on `rocket-scripts` side.

### PR DESCRIPTION
**Pull request by Khadreal**
# Description
Update rocket script

Fixes #6728, #6959, #6978, #6981

## Type of change

- [x] Enhancement (non-breaking change which improves an existing functionality).

## Detailed scenario
N/A

## Technical description
Release version 1.0.8

### Documentation

*Explain how this code works. Diagrams & drawings are welcome.*

# Mandatory Checklist

## Code validation

- [x] I validated all the Acceptance Criteria. If possible, provide screenshots or videos.
- [x] I triggered all changed lines of code at least once without new errors/warnings/notices.
- [x] I implemented built-in tests to cover the new/changed code.

## Code style
- [x] I wrote a self-explanatory code about what it does.
- [x] I protected entry points against unexpected inputs.
- [x] I did not introduce unnecessary complexity.
- [x] Output messages (errors, notices, logs) are explicit enough for users to understand the issue and are actionnable.

### CODE REVIEW

**Comment by Mai-Saad** [comment]
e2e results using wpr zip generated from PR:
-  lcp e2e results on develop commit id  `7b589ba20f78d9f25d0d3edaec1278cf60f92b33` 
[lcp_rocketscriptupdate.pdf](https://github.com/user-attachments/files/17414733/lcp_rocketscriptupdate.pdf)

Rerunning failed the test only on develop using commit id `1a6694c5c2277bb7a59154a8faee43d2dd6b159b` and it's working 
[rocketscriptrerunfailedtest.pdf](https://github.com/user-attachments/files/17414731/rocketscriptrerunfailedtest.pdf)

- Remain, sharing results of lrc e2e using the current PR 
LRC is green (5 templates pass after increase time here to 5000 https://github.com/wp-media/wp-rocket-e2e/blob/develop/backstop_data/engine_scripts/wait.js#L2
[lrcVR.pdf](https://github.com/user-attachments/files/17436240/lrcVR.pdf)

**Comment by MathieuLamiot** [comment]
@Mai-Saad I am not sure to understand your comment. Can you rephrase? Thanks

**Review by Miraeld — APPROVED** [review]

**Comment by Mai-Saad** [comment]
@MathieuLamiot sorry if it wasnot clear, I rephrased the text above which is mainly sharing results of the current PR lcp e2e results (I had to run on 2 different commit id of e2e, as delete plugin isnot working on latest develop of e2e)

**Comment by MathieuLamiot** [comment]
Does this mean there is a regression @Mai-Saad ?

**Comment by Mai-Saad** [comment]
@MathieuLamiot PR is good, regression is at e2e.
